client/src/components/Framework/data/markup-strings.js

```diff
@@ -0,0 +1,49 @@
+export let whiteHTML = `<div className="white-card">
+<div className="white-card-body">
+    <!-- JSON card content -->
```

Comment:
Ive been looking at a way to do this better but it looks quite difficult. I might take a look myself but for now I think we should just leave it as is :)